recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0-3" %}
+
+package:
+    name: r-gsw
+    version: {{ version | replace('-', '_') }}
+
+source:
+    fn: gsw_1.0-3.tar.gz
+    url:
+        - http://cran.r-project.org/src/contrib/gsw_{{ version }}.tar.gz
+        - http://cran.r-project.org/src/contrib/Archive/gsw/gsw_{{ version }}.tar.gz
```

Comment:
The double url is because when a package is updated, cran automatically archives the previous version. Obviously, this would break a number of recipes because the URL would no longer be valid.  That is why there are two URLs.